### PR TITLE
Add MonoDevelop.Sdk package to the mirror.

### PR DESCRIPTION
**Pull request by NTaylorMullen**

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -75,6 +75,7 @@
     <MicrosoftWebXdtPackageVersion>1.4.0</MicrosoftWebXdtPackageVersion>
     <MicrosoftWin32RegistryPackageVersion>4.5.0-preview1-25914-04</MicrosoftWin32RegistryPackageVersion>
     <MonoAddinsPackageVersion>1.3.7</MonoAddinsPackageVersion>
+    <MonoDevelopSdkPackageVersion>1.0.0</MonoDevelopSdkPackageVersion>
```
Comment (NTaylorMullen) [inline build/dependencies.props]: Given that the package name is `MonoDevelop.Sdk` will this break any of our pascal casing code?

Comment (natemcmaster) [reply build/dependencies.props]: No, it should be fine. You can test this locally by running `build.cmd /t:GeneratePropsFiles`. Look in `obj/` for the file it creates.

**Comment by natemcmaster** [comment]
cc @Eilon. Does MonoDevelop count as a third-party dependency?

**Comment by NTaylorMullen** [comment]
> Sure. Could you link the staged PR that will be using this?

https://github.com/aspnet/Razor/pull/1790

File: build/dependencies.props (hunk 1 above)
Comment (muratg) [reply build/dependencies.props]: Just curious. Why do we need this?

Comment (NTaylorMullen) [reply build/dependencies.props]: > Just curious. Why do we need this?

This is for VS for mac Razor support.

**Comment by Eilon** [comment]
> cc @Eilon. Does MonoDevelop count as a third-party dependency?

It's not 3rd party, but what exactly is the concern?

**Comment by natemcmaster** [comment]
OSS approval.

**Comment by natemcmaster** [comment]
:shipit: I'm assuming we don't need OSS approval since this doesn't count as a 3rd party dependency.

**Pull request merged**